chrome/dist/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Local Home SDK App</title>
+  <script type="module" src="main.js"></script>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
```

Comment:
what about building this a series of https://lit-element.polymer-project.org/ ?

Reply:
I opened issue #63 since I think that's a great idea but want to limit the scope of this PR.